docs/en/faq.md

```diff
@@ -2,238 +2,247 @@
     * Map post ID and LucaS video ID (convenient for the players coming from LucaS videos).
     * Related videos for previewing maps.
     * etc...
 
-    **See [pseudoacetyl/BallanceCustomMaps](https://github.com/pseudoacetyl/BallanceCustomMaps/blob/main/CustomMapList.md) for more infomation.** Thank pseudoacetyl!
+    **See [pseudoacetyl/BallanceCustomMaps](https://github.com/pseudoacetyl/BallanceCustomMaps/blob/main/CustomMapList.md) for more infomation.**
+    Huge shoutouts to Pseudoacetyl for the enormous amount of effort spent making the aforementioned table!
 
 ### Map Pack
 
-Map Pack contains lots of high-quality custom maps. However, the map pack may only be updated every few years. So if you need some latest custom maps, try other methods.
+The Ballance Map Pack contains lots of high-quality custom maps. However, the map pack's update are very few and far between, so if you want some new custom maps, try the other methods.
 
-__[**Click Here to Download Ballance Map Pack (Version 7.6)**](http://www.mediafire.com/folder/vyzo7893i0oln/Ballance_Map_Packs)__
+__[**Click here to download the Ballance Map Pack (Version 7.6)**](http://www.mediafire.com/folder/vyzo7893i0oln/Ballance_Map_Packs)__
 
 ### Discord
 
-You can find recent custom maps in `#new`. You also can search `#new` with file limitation to get what you need.  
-However, this Discord server was established in 2018, so obviously some custom maps can not be found in our Discord server channels. In this case, try using other methods.
+You can find recent custom maps in `#news`. You also can search in `#news` with the `has:file` filter to narrow down search results.  
+However, this Discord server was established in 2018, so obviously some custom maps aren't included in our Discord server channels. In this case, try using other methods.
 
-### Network Disk
+### Chinese Network Disk
 
-Our network disk allow you download custom maps individually. It also contain the oldest and the latest custom maps. However, this network disk is hosted by Chinese Ballance Forum, translators needed.
+Our Chinese network disk allows you download custom maps individually.  
+It also contains some of the oldest and the latest custom maps.  
+However, this network disk is hosted by the Chinese Ballance Forum, and we currently lack translators.
 
-[Click here to download separate map files at our Network disk](http://ballancemaps.ys168.com)  
+__[**Click here to visit the Chinese network disk**](http://ballancemaps.ys168.com)__
 
 ## <span id="jump-8">Recording Ballance Videos</span>
 
-Want to record your ballance runs but don't know how to?
-
-Well, don't worry, there is always a solution. Download OBS Studio here: https://obsproject.com/
+[Download OBS Studio.](https://obsproject.com/)  
 
-It supports MP4, youtube, and more.
+It supports MP4, YouTube, and more.  
 
-Tutorial video: https://m.youtube.com/watch?v=XllRHuCe07A
+Tutorial video: https://m.youtube.com/watch?v=XllRHuCe07A  
```

Comment:
2 white space tail is only necessary when you want to create a new line without creating a double line breaker in markdown. these white space are unnecessary.

```
# OK

foo bar<white space><white space>
foo bar

# Unnecessary

foo bat<white space><white space>

foo bar
```

additionally, hope you can change this mobile page url to desktop page url. although youtube have redirect.